kokoro/oak_containers_presubmit.sh

```diff
@@ -25,6 +25,7 @@ echo "${KOKORO_GIT_COMMIT_oak:?}" > "$KOKORO_ARTIFACTS_DIR/binaries/git_commit"
 # Copy the generated binaries to Placer.
 export GENERATED_BINARIES=(
     ./target/stage1.cpio
+    ./target/vmlinux-6.1.33
```

Comment:
I think this may be in a different subfolder?

Reply:
There are too many `target` directories.